File: APEC.RRHH.api/Core/Models/CandidateEmployeeAspiratedJob.cs
using System;
using Core.Contracts;
using Core.Enums;

namespace Core.Models
{
    public sealed class CandidateEmployeeAspiratedJob : IEntityBase
    {
        public Guid Id { get; set; }
        public Guid CandidateEmployeeId { get; set; }
        public CandidateEmployee CandidateEmployee { get; set; }
        public Guid JobId { get; set; }
        public Job Job { get; set; }
        public FeatureStatus Status { get; set; }
        public int SalaryToAspire { get; set; }
        public Guid? UserIdWhoRecommendedIt { get; set; }
        public User UserWhoRecomendedIt { get; set; }
    }
}

File: APEC.RRHH.webapp/src/app/models/candidate-employee-aspirated-job.ts
import { UserProfile } from './user-profile';
import { Job } from './job';

export class CandidateEmployeeAspiratedJob {
    id: string;
    candidateEmployeeId: string;
    jobId: string;
    job: Job;
    salaryToAspire: number;
    userIdWhoRecommendedIt: string;
    userWhoRecommendedIt: UserProfile;
}
